app/src/main/res/layout/message_item.xml

```diff
@@ -27,7 +27,7 @@
             app:srcCompat="@android:color/darker_gray" />
 
     <TextView
-            android:id="@+id/text_subject"
+            android:id="@+id/subject"
             android:layout_width="253dp"
             android:layout_height="63dp"
             android:layout_alignParentTop="true"
```

Comment:
Margins are all off, please arrange all the views in this file, otherwise, the information is going to be hidden

Reply:
i change them but i don know if they are correct